### DISCUSSION TITLE
Moving to different organization

### DISCUSSION THREAD

**Discussion by scaronni** [dkms-project/dkms#251] [Q&A]
Now that administrator access has been restored, we can finally make changes to the repository, like adding new contributors, renaming, planning, etc.

The project is not affiliated with Dell in any ways, so I think it should be hosted in a different organization, whether its own or a pre-existing one.

What are the thoughts and preferences on this?

**Comment by evelikov** [comment]
As mentioned in https://github.com/dell/dkms/issues/249 I see a new org as a good way forward. In a gist it provides clarity and direction about the (current) people behind the project plus it allows segregating bits (say templates or other) and having broader developer base behind them.

**Reply by scaronni** [reply]
So DKMS with under it `dkms` as the main repository?

Separate templates as your suggeston? Like `template_Debian`, `template_Arch`, `template_RedHat`?

**Reply by evelikov** [reply]
Yes in general, but bear in mind - the template is just an example. As time goes on, we could add a repo with - say docker containers/recipes - or anything else that seems relevant.
